src/live-common-setup.js

```diff
@@ -51,6 +52,7 @@ setSupportedCurrencies([
   "bitcoin_testnet",
   "ethereum_ropsten",
   "cosmos_testnet",
+  "crypto_org_croeseid",
   "elrond",
 ]);
```

Comment:
Can you not enable the coin in this PR ? thanks

Reply:
Done